### PR TITLE
fix: correct TreeBreadcrumb root on Windows ICIJ/datashare#1012

### PR DESCRIPTION
**Pull request by mvanzalu**

### CODE REVIEW

File: src/components/TreeBreadcrumb.vue [hunk 1]
```diff
@@ -74,13 +74,23 @@ export default {
       default: false
     }
   },
+  data() {
+    return {
+      pathSeparator: this.$core.config.get('pathSeparator')
```
Comment (caro3801) [inline src/components/TreeBreadcrumb.vue:79]: What happen if the pathSeparator is not configured (or wrongly defined)? do we have a fallback?

**Review by caro3801 — CHANGES_REQUESTED** [review]
Adding a test with a different (windows?) separator to document the fix would be a great addition

File: src/components/TreeBreadcrumb.vue (hunk 1 above)
Comment (pirhoo) [reply src/components/TreeBreadcrumb.vue:79]: `$config.get` accepts a default value: `this.$core.config.get('pathSeparator', '/')`

Comment (mvanzalu) [reply src/components/TreeBreadcrumb.vue:79]: Good remark

**Review by caro3801 — APPROVED** [review]

**Pull request merged**